recipes/gstreamer/all/patches/150.diff

```diff
@@ -0,0 +1,34 @@
+diff --git a/gst/meson.build b/gst/meson.build
+index 145e97a89f1df5436a5014a73828b5e02b2fcaa3..d19689c431521662215c8d91ba37dcd3974e4e1b 100644
+--- a/gst/meson.build
++++ b/gst/meson.build
+@@ -237,6 +237,7 @@ gst_incdirs = [configinc]
+ gst_gen_sources = [gstenum_h]
+ libgst = library('gstreamer-1.0', gst_sources,
+   gstenum_h, gstenum_c, gst_parse_sources, gst_registry_sources,
++  objects : printf_lib.extract_all_objects(),
+   version : libversion,
+   soversion : soversion,
+   darwin_versions : osxversion,
+@@ -244,7 +245,6 @@ libgst = library('gstreamer-1.0', gst_sources,
+   include_directories : [configinc,
+     # HACK, change include paths in .y and .l in final version.
+     include_directories('parse')],
+-  link_with : printf_lib,
+   install : true,
+   dependencies : [gobject_dep, gmodule_dep, glib_dep, mathlib, dl_dep] + backtrace_deps
+                    + platform_deps,
+@@ -275,7 +275,13 @@ if build_gir
+   )]
+ endif
+ 
++gst_compile_args = []
++if get_option('default_library') == 'static'
++  gst_compile_args += ['-DGST_STATIC_COMPILATION']
++endif
```

Comment:
If this flag is added in a patch why is it exposed in the package info? (i am not able to find it _quickly_ in the docs)

Reply:
because this macro needs to be defined both when building and consuming gstreamer https://lists.freedesktop.org/archives/gstreamer-commits/2016-June/094858.html

Reply:
that said, the patch is already upstreamed !